src/client.rs

```diff
@@ -246,20 +244,26 @@ impl Client {
         if let Ok(mut sync) = sync.write() {
             *sync = synced_nodes;
         }
+
+        network_info.write().map_err(|_| crate::Error::PoisonError)?.synced = true;
```

Comment:
@Thoralf-M Should this be done all the time or only if `synced_nodes` is not empty ?

Reply:
Only if it's not empty

Reply:
Found it